examples/purchase-tester/build.gradle

```diff
@@ -72,4 +72,5 @@ dependencies {
 
     implementation "androidx.navigation:navigation-fragment-ktx:2.5.2"
     implementation "androidx.navigation:navigation-ui-ktx:2.5.2"
+    implementation project(path: ':feature:google')
```

Comment:
This shouldn't be needed. Why did you add this?

Reply:
Purchase Tester tries to call `storeProduct.googlePrduct` to show some detail and because `GoogleStoreProduct` was moved into the `google` module, this is now needed. That is the only reason it is required though!